src/__tests__/fake-timers.js

```diff
@@ -48,13 +48,10 @@ test('times out after 1000ms by default', async () => {
   await expect(
     waitForElementToBeRemoved(() => container, {onTimeout: e => e}),
   ).rejects.toThrowErrorMatchingInlineSnapshot(
-    `"Timed out in waitForElementToBeRemoved."`,
+    `Timed out in waitForElementToBeRemoved.`,
   )
-  // NOTE: this assertion ensures that the timeout runs in the declared (fake) clock
-  // while in real time the time was only a fraction since the real clock is only bound by the CPU
-  // So 10ms is really just an approximation on how long the CPU needs to execute our code.
-  // If people want to timeout in real time they should rely on their test runners.
-  expect(performance.now() - start).toBeLessThanOrEqual(10)
+  // NOTE: this assertion ensures that the timeout runs in the declared (fake) clock.
+  expect(performance.now() - start).toBeGreaterThanOrEqual(1000)
```

Comment:
So, just a question for me...
This change is because `perfomance` is now also affected when we advance the time?

Reply:
This was my interpretation as well. I'm going to check again, just in cae.

Reply:
Separated the assertions into fake and real runtime so that's explicit how `timeout` behaves: [`2632aae` (#962)](https://github.com/testing-library/dom-testing-library/pull/962/commits/2632aae7bbf8804bd5800b674572c33835afe328)

Reply:
Great, thanks @eps1lon ! 
This LGTM.